configure.ac

```diff
@@ -202,6 +214,31 @@ eval oldincludedir="`eval echo ${oldincludedir}`"
 eval infodir="`eval echo ${infodir}`"
 eval mandir="`eval echo ${mandir}`"
 
+dnl Home-grown variables
+eval INITDIR="${INITDIR}"
+AC_MSG_NOTICE(Sanitizing INITDIR: ${INITDIR})
+case $INITDIR in
+  prefix) INITDIR=$prefix;;
```

Comment:
We're planning to remove the above bit from pacemaker. Probably best not to add it from the beginning in sbd.